lib/App.js

```diff
@@ -65,6 +64,25 @@ App.prototype._init = function() {
   this._waitForAttach = true;
   this._cancelAttach = false;
   this.model = new this.derby.Model();
+  const appname = this.name;
+
+  // import(viewImport).then(serializedViews => {
+  //   console.log(serializedViews);
+  //   serializedViews(derbyTemplates, this.views);
+  //   // Must init async so that app.on('model') listeners can be added.
+  //   // Must also wait for content ready so that bundle is fully downloaded.
+  //   this._contentReady();
+  // })
+
+  // require.ensure([`derby/lib/${appname}__views`]).then(require => {
+  //   const serializedViews = requrie(`derby/lib/${appname}__views`)
+  //   serializedViews(derbyTemplates, this.views);
+  //   // Must init async so that app.on('model') listeners can be added.
+  //   // Must also wait for content ready so that bundle is fully downloaded.
+  //   this._contentReady();
+  // });
+
+  const serializedViews = require(`derby/lib/${appname}__views`);
```

Comment:
1. This would be a breaking change for derby-browserify and would necessitate a new major version. So either we do that, or put this behind a constructor/plugin option so that derby-webpack can enable the prefixed view names while leaving derby-browserify as-is.
2. This repo's still using ES5 syntax, so for consistency change this to `var` and use string concat instead of interpolation